Libraries/Opc.Ua.Server/Subscription/MonitoredItem.cs

```diff
@@ -30,10 +30,23 @@
 using System;
 using System.Collections.Generic;
 using System.Globalization;
+using System.Threading;
 using System.Xml;
 
 namespace Opc.Ua.Server
 {
+    /// <summary>
+    /// Specifies the values of the ResendData state for a monitored item
+    /// The state is set on the ResendData method call
+    /// </summary>
+    internal enum ResendDataState : int
+    {
```

Comment:
why is an enum defined but the class uses an int for the variable?

Reply:
Due to the interlock usage. This will change since using interlock is useless.